cmake/generic_plugin.cmake

```diff
@@ -1,7 +1,21 @@
 function (create_generic_plugin generic_plugin_name)
+
 	set(PROJECT_NAME ${generic_plugin_name})
+
+	include(CheckIPOSupported)
```

Comment:
acum nu cred ca mai trebuie asta in cmakeul de pluginuri. cmakeul principal ar trebui sa acopere tot

Reply:
done